src/test/java/core/basesyntax/RegistrationTests.java

```diff
@@ -0,0 +1,145 @@
+                + " to be thrown for the null user login, but it wasn't");
+        System.out.println("Test register_user_with_empty_login_NotOk passed");
+    }
+
+    @Test
+    @DisplayName("Registration an user with null age")
+    public void register_user_with_null_age_NotOk() {
+        invalidUser.setAge(null);
+        assertThrows(RegistrationIllegalArgumentException.class, () -> {
+            registrationService.register(invalidUser); }, "Expected "
+                + RegistrationIllegalArgumentException.class.getName()
+                + " to be thrown for user with null age, but it wasn't");
+        System.out.println("Test register_user_with_null_age_NotOk passed");
+    }
+
+    @Test
+    @DisplayName("Registration an user with nonacceptable age")
+    public void register_user_with_nonacceptable_age_NotOk() {
+        assertThrows(RegistrationIllegalArgumentException.class, () -> {
+            registrationService.register(invalidUser); }, "Expected "
+                + RegistrationIllegalArgumentException.class.getName()
+                + " to be thrown for user with nonacceptable age, but it wasn't");
+        System.out.println("Test register_user_with_nonacceptable_age_NotOk passed");
+    }
+
+    @Test
+    @DisplayName("Registration an user with null password")
+    public void register_user_with_null_password_NotOk() {
+        invalidUser.setPassword(null);
+        assertThrows(RegistrationIllegalArgumentException.class, () -> {
+            registrationService.register(invalidUser); }, "Expected "
+                + RegistrationIllegalArgumentException.class.getName()
+                + " to be thrown for user with null password, but it wasn't");
+        System.out.println("Test register_user_with_null_password_NotOk passed");
+    }
+
+    @Test
+    @DisplayName("Registration an user with nonacceptable length of password")
+    public void register_user_with_nonacceptable_password_NotOk() {
+        assertThrows(RegistrationIllegalArgumentException.class, () -> {
+            registrationService.register(invalidUser); }, "Expected "
+                + RegistrationIllegalArgumentException.class.getName()
+                + " to be thrown for user with nonacceptable length of password, but it wasn't");
+        System.out.println("Test register_user_with_nonacceptable_password_NotOk passed");
+    }
+
+    @Test
+    @DisplayName("Repeatedly registration an user that was registered before")
+    public void register_existing_user_again_NotOk() {
+        Storage.people.clear();
+        storageDao.add(validUser);
+        assertThrows(RegistrationIllegalArgumentException.class, () -> {
+            registrationService.register(validUser); }, "Expected "
+                + RegistrationIllegalArgumentException.class.getName()
+                + " to be thrown for existing into Storage user, but it wasn't");
+        System.out.println("Test register_existing_user_again_NotOk passed");
+    }
+
```

Comment:
Please add test for successful registration of new user, and check if added user equals to one returned from method registrationService.register(validUser);

Reply:
ok
i added suitable test